hack/build/config.sh

```diff
@@ -61,6 +61,9 @@ QUAY_REPOSITORY=${QUAY_REPOSITORY:-cdi-operatorhub}
 QUAY_NAMESPACE=${QUAY_NAMESPACE:-kubevirt}
 CDI_LOGO_PATH=${CDI_LOGO_PATH:-"assets/cdi_logo.png"}
 
+# oVirt populator image from forklift
+OVIRT_POPULATOR_IMAGE_NAME=${OVIRT_POPULATOR_IMAGE_NAME:-"quay.io/kubev2v/ovirt-populator:release-v2.5.4"}
```

Comment:
cc @alromeros @mhenriks 
my worry is productizing this in downstream

Reply:
WDYT about including & using the sources under CDI? at least in our scripts
On other distributions (downstream) the env var will be set to something else and that will get tested instead.

Reply:
no problem

Reply:
this is not implemented yet, right?

Reply:
not yet, just to clarify, we want to build it in CDI?
something like was done here: https://github.com/kubevirt/containerized-data-importer/pull/2895 ?

Reply:
yeah, any drawbacks? lets us stay completely in charge upstream if anything breaks down

Reply:
not sure why you linked the closed PR, the one that merged also had the ovirt populator sources in it:
https://github.com/kubevirt/containerized-data-importer/pull/2947

Reply:
in the closed one ovirt-populator is built separately which is what I assume we want here, while in https://github.com/kubevirt/containerized-data-importer/pull/2947 it's part of cdi-importer

Reply:
ah, I see. maybe for consistency let's keep it non separate? wdyt?

Reply:
published, only did a basic sanity check but looks ok